workspaces/liferay-paypal-workspace/client-extensions/liferay-paypal-batch/batch/object-definition.batch-engine-data.json

```diff
@@ -0,0 +1,204 @@
+{
+	"configuration": {
+		"className": "com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition",
+		"parameters": {
+			"containsHeaders": "true",
+			"createStrategy": "UPSERT",
+			"importStrategy": "ON_ERROR_FAIL",
+			"updateStrategy": "UPDATE"
+		},
+		"taskItemDelegateName": "DEFAULT"
+	},
+	"items": [
+		{
+			"accountEntryRestricted": false,
+			"accountEntryRestrictedObjectFieldName": "",
+			"active": true,
+			"defaultLanguageId": "en_US",
+			"enableCategorization": true,
+			"enableComments": false,
+			"externalReferenceCode": "N2A1_PAYPAL_TRANSACTION",
```

Comment:
what is N2A1?

Reply:
@luca-pellizzon , I talked to Crescenzo about this. He said the externalReferenceCode was something we choose on our own. I just followed his format and added +1 to it. His format was n1a0

Reply:
that's a webhook code in Adyen, if there's no N2A1 in PayPal it doesn't make any sense

Reply:
Okay, that makes more sense to me because the format did seem arbitrary. If it is a webhook code in Adyen, that is a pattern that I should definitely not use. I will remove the pattern entirely and just use `c_paypaltransaction` for example.